scripts/e2etests/NuGetFunctionalTestUtils.ps1

```diff
@@ -215,36 +210,30 @@ function CopyResultsToCI
     $RealTimeResultsFilePath = Join-Path $ResultsFileParent.FullName 'Realtimeresults.txt'
 
     $TestResultsPath = Join-Path $DropPathParent.FullName 'testresults'
+    $FullLogFileDestinationPath = Join-Path $TestResultsPath $FullLogFileName
     mkdir $TestResultsPath -ErrorAction Ignore
 
     $DestinationFileName = 'Run-' + $RunCounter + '-' + (Split-Path $resultsFile -Leaf)
     $DestinationPath = Join-Path $TestResultsPath $DestinationFileName
-    Write-Host "Copying results file from $resultsFile to $DestinationPath"
+    Write-Host "Copying html results file from $resultsFile to $DestinationPath"
     Copy-Item $resultsFile $DestinationPath
-    if($env:CI -and $env:EndToEndResultsDropPath)
+    if($env:CI)
     {
-        Write-Host "Copying full log file from $FullLogFilePath to $env:EndToEndResultsDropPath"
-        if(-not (Test-Path $env:EndToEndResultsDropPath))
-        {
-            New-Item -Path $env:EndToEndResultsDropPath -ItemType Directory -Force
-        }
-        Copy-Item $FullLogFilePath -Destination $env:EndToEndResultsDropPath -Force  -ErrorAction SilentlyContinue
-
-        Write-Host "Copying test results file from $resultsFile to $env:EndToEndResultsDropPath"
-        Copy-Item $resultsFile -Destination $env:EndToEndResultsDropPath -Force -ErrorAction SilentlyContinue
+        Write-Host "Copying full log file from $FullLogFilePath to $FullLogFileDestinationPath"
+        Copy-Item $FullLogFilePath -Destination $FullLogFileDestinationPath -Force  -ErrorAction SilentlyContinue
     }
 
     OutputResultsForCI -NuGetDropPath $NuGetDropPath -RunCounter $RunCounter -RealTimeResultsFilePath $RealTimeResultsFilePath
 }
 
-function CopyActivityLogToCI
-{    
-    if($env:ActivityLogFullPath) 
-    {
-        Write-Host "Copying activity log file from $env:ActivityLogFullPath to $env:EndToEndResultsDropPath"
-        Copy-Item $env:ActivityLogFullPath -Destination $env:EndToEndResultsDropPath -Force  -ErrorAction SilentlyContinue
-    }
-}
+# function CopyActivityLogToCI
```

Comment:
why did we comment this out?

Reply:
hasn't really been helpful, and don't want to spend too much time trying to make this work. will un-comment if we find a need for it later.